graylog2-server/src/main/java/org/graylog/plugins/views/search/rest/SearchResource.java

```diff
@@ -203,6 +204,8 @@ public Response executeQuery(@ApiParam(name = "id") @PathParam("id") String id,
                 throwStreamAccessForbiddenException();
             }
 
+            // Unless explicitly queried, exclude event indices by default
+            allAvailableStreamIds.removeAll(DEFAULT_EVENT_STREAM_IDS);
```

Comment:
Please extend the comment to describe why we are doing this and also add the link to https://github.com/Graylog2/graylog2-server/issues/6490 as a `TODO` so we know that it can be removed once that issue got implemented. Thanks!